mocks.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2019 The Prometheus Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tsdb
+
+import (
+	"context"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+
+	"github.com/prometheus/tsdb/labels"
+	"github.com/prometheus/tsdb/testutil"
+	"github.com/prometheus/tsdb/tsdbutil"
+)
+
+// This file holds types and functions that are used for testing
+// purposes.
+
+const (
+	MockDefaultLabelName  = "labelName"
+	MockDefaultLabelValue = "labelValue"
+)
+
+type mockSeries struct {
+	labels   func() labels.Labels
+	iterator func() SeriesIterator
+}
+
+// MockCreateBlock creates a block with given set of series and returns its dir.
+// Intended for testing purposes.
+func MockCreateBlock(tb testing.TB, dir string, series []Series) string {
```

Comment:
I prefer the original name.  just need to find a good package name or put it one of the existing packages.

```suggestion
func CreateBlock(tb testing.TB, dir string, series []Series) string {
```

Reply:
Sure, I renamed it back to `CreateBlock`.